docker-compose.yml

```diff
@@ -102,7 +102,15 @@ services:
         - COUCHDB_PASSWORD=123456
         - COUCHDB_USER=admin
      ports:
-        - 5986:5986       
-         
+        - 5986:5986   
+        
+  pad-b-tracing: 
+     build: ../tracing/
+     depends_on:
+       - pad-b-registry
+     ports:

```

Comment:
We are using the 8000 porte range, I think

Reply:
Yep, we are using this range on the others, from 8080 increasing 1 in each service.

Reply:
Maybe we can make tracing run on 8090?

Reply:
ok